specify/specify7
# Check isBulkCarry config  for nested relationships

Fixes #5205

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- verify that count field is required in schema config for prep table 
- Create a new CO record
- Click on the gear icon to access the form meta
- Enable the "Show Carry Forward Button" and "Show bulk carry forward count"
- Click on the gear icon next to "Show Carry Forward Button"
- Uncheck a required field in a nested relationship, ex count in prep 
- Fill out required fields and save the record
- Input a value higher than 1 for "Bulk Carry Forward Count"
- Click "Carry Forward"
- [ ] verify the clone works and the amount of prep as been carried away for the bulk
